include/Ecal/EcalVetoProcessor.h

```diff
@@ -94,6 +103,12 @@ class EcalVetoProcessor : public framework::Producer {
 
   void buildBDTFeatureVector(const ldmx::EcalVetoResult& result);
 
+  // MIP tracking
+  /* Function to find distance between two lines (line 1 passing through v1 and v2, etc.) */
```

Comment:
Fix the function documentation so it shows up correctly when the documentation is built.

Reply:
Just to confirm, this only involves changing the single-line comments to multi-line comments while leaving them where they are, right?

Reply:
Yes you leave them where they are, but the format is
```
/**
 * short description
 *
 * longer 
 * multi-line 
 * description
 *
 * @param[in] param1 description of param1
 * @param[in] param2 description of param2
 * ..other params
 * @returns description of return value if applicable
 */
```